splash/cookies.py

```diff
@@ -78,23 +78,23 @@ def add(self, cookie):
     @classmethod
     def har_cookie2qt(cls, cookie):
         qcookie = QNetworkCookie()
-        qcookie.setName(cookie["name"])
-        qcookie.setValue(cookie["value"])
+        qcookie.setName(cookie[b"name"])
+        qcookie.setValue(cookie[b"value"])
```

Comment:
It makes more sense to use unicode for key names, unless it is hard for some reason.